packages/xsnap/test/test-xsnap.js

```diff
@@ -349,3 +349,17 @@ test('normal close of pathological script', async t => {
   await vat.terminate();
   await hang;
 });
+
+test('heap exhaustion: orderly fail-stop', async t => {
+  const grow = `
+  let stuff = '1234';
+  while (true) {
+    stuff = stuff + stuff;
+  }
+  `;
+  for await (const debug of [false, true]) {
```

Comment:
No need for `for await...of` here.  This isn't an async iterable.

```suggestion
  for (const debug of [false, true]) {
```

Reply:
I got a red squiggle when I wrote it that way... I thought it was a correctness thing, but now I see it's https://eslint.org/docs/rules/no-await-in-loop ... that recommends `Promise.all`. hm. I guess there's no data dependency here, but intermixed logs can be confusing. I suppose that argues for splitting this into 2 tests to let ava manage the parallelism and correlating logs. hm. Maybe next time...